{
    "data_version": "4.0",
    "data_type": "CVE",
    "data_format": "MITRE",
    "CVE_data_meta": {
        "ID": "CVE-2025-3730",
        "ASSIGNER": "cna@vuldb.com",
        "STATE": "PUBLIC"
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "** DISPUTED ** A vulnerability, which was classified as problematic, was found in PyTorch 2.6.0. Affected is the function torch.nn.functional.ctc_loss of the file aten/src/ATen/native/LossCTC.cpp. The manipulation leads to denial of service. An attack has to be approached locally. The exploit has been disclosed to the public and may be used. The real existence of this vulnerability is still doubted at the moment. The name of the patch is 46fc5d8e360127361211cb237d5f9eef0223e567. It is recommended to apply a patch to fix this issue. The security policy of the project warns to use unknown models which might establish malicious effects."
            },
            {
                "lang": "deu",
                "value": "** DISPUTED ** Es wurde eine Schwachstelle in PyTorch 2.6.0 gefunden. Sie wurde als problematisch eingestuft. Hiervon betroffen ist die Funktion torch.nn.functional.ctc_loss der Datei aten/src/ATen/native/LossCTC.cpp. Mit der Manipulation mit unbekannten Daten kann eine denial of service-Schwachstelle ausgenutzt werden. Der Angriff hat dabei lokal zu erfolgen. Der Exploit steht zur \u00f6ffentlichen Verf\u00fcgung. Die wahre Existenz der vermeintlichen Schwachstelle wird zur Zeit in Frage gestellt. Der Patch wird als 46fc5d8e360127361211cb237d5f9eef0223e567 bezeichnet. Als bestm\u00f6gliche Massnahme wird Patching empfohlen."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "Denial of Service",
                        "cweId": "CWE-404"
                    }
                ]
            }
        ]
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "n/a",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "PyTorch",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_affected": "=",
                                            "version_value": "2.6.0"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "references": {
        "reference_data": [
            {
                "url": "https://vuldb.com/?id.305076",
                "refsource": "MISC",
                "name": "https://vuldb.com/?id.305076"
            },
            {
                "url": "https://vuldb.com/?ctiid.305076",
                "refsource": "MISC",
                "name": "https://vuldb.com/?ctiid.305076"
            },
            {
                "url": "https://vuldb.com/?submit.553645",
                "refsource": "MISC",
                "name": "https://vuldb.com/?submit.553645"
            },
            {
                "url": "https://github.com/pytorch/pytorch/issues/150835",
                "refsource": "MISC",
                "name": "https://github.com/pytorch/pytorch/issues/150835"
            },
            {
                "url": "https://github.com/pytorch/pytorch/pull/150981",
                "refsource": "MISC",
                "name": "https://github.com/pytorch/pytorch/pull/150981"
            },
            {
                "url": "https://github.com/pytorch/pytorch/issues/150835#issue-2979082232",
                "refsource": "MISC",
                "name": "https://github.com/pytorch/pytorch/issues/150835#issue-2979082232"
            },
            {
                "url": "https://github.com/timocafe/tewart-pytorch/commit/46fc5d8e360127361211cb237d5f9eef0223e567",
                "refsource": "MISC",
                "name": "https://github.com/timocafe/tewart-pytorch/commit/46fc5d8e360127361211cb237d5f9eef0223e567"
            }
        ]
    },
    "credits": [
        {
            "lang": "en",
            "value": "Default436352 (VulDB User)"
        }
    ],
    "impact": {
        "cvss": [
            {
                "version": "3.1",
                "baseScore": 3.3,
                "vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L",
                "baseSeverity": "LOW"
            },
            {
                "version": "3.0",
                "baseScore": 3.3,
                "vectorString": "CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L",
                "baseSeverity": "LOW"
            },
            {
                "version": "2.0",
                "baseScore": 1.7,
                "vectorString": "AV:L/AC:L/Au:S/C:N/I:N/A:P"
            }
        ]
    }
}